circle.yml

```diff
@@ -6,19 +6,21 @@ dependencies:
   override:
     - pip install --upgrade pip
     - pip install sphinx matplotlib coverage Pillow
-    - pip install sphinx-gallery sphinx_bootstrap_theme
+    - git clone https://github.com/sphinx-gallery/sphinx-gallery.git
+    - cd sphinx-gallery && git fetch --tags && git fetch --tags && git checkout v0.1.11
```

Comment:
why twice git fetch --tags ?

Reply:
oops, good point. Fixed directly in master